testing-modules/rest-assured/src/test/java/com/baeldung/restassured/RestAssured2IntegrationTest.java

```diff
@@ -24,26 +26,25 @@ public class RestAssured2IntegrationTest {
     private static final String ODDS = getJson();
 
     @BeforeClass
-    public static void before() throws Exception {
+    public static void before() {
         System.out.println("Setting up!");
         final int port = Util.getAvailablePort();
         wireMockServer = new WireMockServer(port);
         wireMockServer.start();
         configureFor("localhost", port);
         RestAssured.port = port;
-        stubFor(get(urlEqualTo(EVENTS_PATH)).willReturn(
-          aResponse().withStatus(200)
-            .withHeader("Content-Type", APPLICATION_JSON)
-            .withBody(ODDS)));
-        stubFor(post(urlEqualTo("/odds/new"))
-            .withRequestBody(containing("{\"price\":5.25,\"status\":1,\"ck\":13.1,\"name\":\"X\"}"))
+        stubFor(com.github.tomakehurst.wiremock.client.WireMock.get(urlEqualTo(EVENTS_PATH))
```

Comment:
FI: Wiremock get() and RestAssured get() were both statically imported, but now there is a collision because with the bump in the version RestAssured also has a get(String) method, thus this change